autopush/endpoint.py

```diff
@@ -320,7 +325,7 @@ def _process_auth(self, result):
             except ValueError:
                 raise VapidAuthException("Invalid Authorization header")
             # if it's a bearer token containing what may be a JWT
-            if auth_type.lower() == 'bearer' and '.' in token:
+            if auth_type.lower() == AUTH_SCHEME and '.' in token:
```

Comment:
We probably want to make this 
   if auth_type.lower() in [AUTH_SCHEME, 'bearer'] and ...
for a few releases. There are a few folks in the wild using 'bearer' and we don't want to break them yet.